fmriprep/interfaces/images.py

```diff
@@ -279,6 +228,93 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class ConformSeriesInputSpec(BaseInterfaceInputSpec):
+    t1w = File(exists=True, mandatory=True,
+               desc='input T1w image')
+
+    target_zooms = traits.List(traits.Float,
+                               desc='Target zoom information')
+    target_shape = traits.List(traits.Int,
+                               desc='Target shape information')
+    target_span = traits.List(traits.Float,
+                              desc='Target span information')
+
+
+class ConformSeriesOutputSpec(TraitedSpec):
+    t1w = File(exists=True, desc='conformed T1w image')
+
+
+class ConformSeries(SimpleInterface):
```

Comment:
Since the series is being mapped over, perhaps "ConformImage" or possibly "Conform"?